src/pages/diagram/dialogs/list-processes/components/form/form.tsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from "react";
+import * as S from "./styles";
+
+interface IPayload {
+  status?: string;
+  nodeId?: string;
+  initialDate: Date | null;
+  finalDate: Date | null;
+}

```

Comment:
Mover para uma pasta types dentro do list process e chamar de IFilter
```suggestion
interface IFilter{
  status?: string;
  nodeId?: string;
  initialDate: Date | null;
  finalDate: Date | null;
}

```